Starting run
Included groups: failing
Running tests: 002_fail, 010_timeout, 015c_verification_fail

002_fail                                                     ....... [Failed ] rc=1, allowed=0
stdout file: xeet.out/002_fail/stdout (empty)
stderr file: xeet.out/002_fail/stderr (empty)

010_timeout                                                  ....... [Failed ]
Command '['sleep', '5']' timed out after 1 seconds

015c_verification_fail                                       ....... [Failed ] Verification failed w/no output

Finished iteration (#0/0)
Failed: 002_fail, 010_timeout, 015c_verification_fail

